src/schema-association-service.ts

```diff
@@ -103,6 +109,20 @@ async function autoDetectSchema(
     if (!(await azureAccountApi.waitForSubscriptions())) {
         logger.log(`Waiting for login`, 'SchemaDetection');
 
+        // Disable 1ESPT schema and delete 1ESPT schema file if user is signed out
+        const schemaUri1ESPTFolder = Utils.joinPath(context.globalStorageUri, '1ESPTSchema');
+        var schemaUri1ESPTFolderExists = fs.existsSync(schemaUri1ESPTFolder.path.substring(1)); // substring(1) to remove the '/' at the beginning of path
```

Comment:
I do believe you want `fsPath` here, which should allow you to skip the substring. On the other hand, is this check even necessary? You can try unconditionally deleting the 1ESPTSchema folder and wrapping it in a try-catch if necessary.

Reply:
removed this code and put in try catch